applications/sasquatch/values-base.yaml

```diff
@@ -285,6 +285,7 @@ rest-proxy:
     topics:
       - lsst.obsenv.action
       - lsst.obsenv.summary
+      - lsst.obsenv.run_branch
```

Comment:
Adding topics here will create them using the KafkaTopic resource in Strimzi, unfortunately the topic operator does not accept "_" in the topic name, I suggest changing to `lsst.obsenv.run-branch` or similar.